spring-cloud-skipper/src/main/java/org/springframework/cloud/skipper/support/DeploymentPropertiesUtils.java

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.skipper.support;
+
+import java.util.ArrayList;
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.springframework.util.StringUtils;
+
+/**
+ * Provides utility methods for formatting and parsing deployment properties.
+ *
+ * @author Eric Bottard
+ * @author Mark Fisher
+ * @author Janne Valkealahti
+ */
+public final class DeploymentPropertiesUtils {
```

Comment:
Maybe we just rename it to PropertiesUtils and remove the `deployment` prefix that is scattered around in various names?